Symfony/CS/Tests/Fixtures/Integration/priority/class_definition,trailing_spaces.test

```diff
@@ -1,22 +0,0 @@
---TEST--
-Integration of fixers: class_definition, trailing_spaces
---CONFIG--
-level=none
-fixers=class_definition, trailing_spaces
---EXPECT--
-<?php
-class Aaa implements
-    Symfony\CS\Tests\Fixer,
-\RFb,
-    \Fcc1,
-\GFdd
-{
-}
-
---INPUT--
-<?php
-class Aaa implements
-    Symfony\CS\Tests\Fixer,\RFb,
-    \Fcc1, \GFdd
-{
-}
```

Comment:
could we recover that test back and put it in `misc` and test it without priority check ?